packages/e2e-tests/specs/experiments/post-editor-template-mode.test.js

```diff
@@ -89,12 +91,14 @@ const createNewTemplate = async ( templateName ) => {
 
 describe( 'Post Editor Template mode', () => {
 	beforeAll( async () => {
+		await activateTheme( 'tt1-blocks' );
```

Comment:
Wouldn't it be enough to activate the plugin for all tests instead of changing the theme?

```suggestion
		await activatePlugin( 'gutenberg-test-block-templates' );
```

Reply:
I guess that would rely less on the Theme's implementation which is good.

Reply:
That's right. I think it's better that way.